readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -87,29 +87,13 @@ function Promo(data) {
     this.div_id = data.div_id || '';
     this.html = data.html || '';
     this.display_type = data.display_type || '';
-
-    // Handler when a promo receives a click
-    this.click_handler = function () {
-        // This needs to handle both old style legacy analytics for previously built docs
-        // as well as the newer universal analytics
-        if (typeof ga !== 'undefined') {
-            ga('rtfd.send', 'event', 'Promo', 'Click', data.id);
-        } else if (typeof _gaq !== 'undefined') {
-            _gaq.push(
-                ['rtfd._setAccount', 'UA-17997319-1'],
-                ['rtfd._trackEvent', 'Promo', 'Click', data.id]
-            );
-        }
-    };
```

Comment:
I wonder about leaving this logic in, until we can properly confirm the server-side stuff is working. Not sure how important it is, if it's better to possible double count or to just lose the data.